### PR TITLE
add sqs delays for restart

### PR DESCRIPTION
**Pull request by apoorvsadana**

### CODE REVIEW

File: src/publishers/mod.rs [hunk 1]
```diff
@@ -4,8 +4,20 @@ use aws_sdk_sqs::Error;
 
 use crate::config::config;
 
-async fn send_sqs_message(queue_url: &str, message: &str) -> Result<(), Error> {
+async fn send_sqs_message_with_delay(queue_url: &str, message: &str, delay_seconds: i32) -> Result<(), Error> {
```
Comment (EvolveArt) [inline src/publishers/mod.rs:7]: can delay be negative ?

**Review by EvolveArt — APPROVED** [review]

File: src/publishers/mod.rs (hunk 1 above)
Comment (apoorvsadana) [reply src/publishers/mod.rs:7]: I don't think so but the aws sdk expects an i32

Comment (EvolveArt) [reply src/publishers/mod.rs:7]: we can pass it as u32 and just cast it in the call then

Comment (apoorvsadana) [reply src/publishers/mod.rs]: sure, fixed!

**Pull request merged**